packages/carbon-graphs/src/js/controls/Line/Line.js

```diff
@@ -256,6 +256,22 @@ class Line extends GraphContent {
         this.dataTarget,
         graph.config,
       );
+    } else if (graph.config.axis.y2.show) {
```

Comment:
is this the same exact logic as the bar? if so, is it possible to extract this into pure function for reuse?

Reply:
Fixed [here](https://github.com/cerner/terra-graphs/pull/94/commits/db69bb1aa18d032ad4a1a98365fe5dc2bb8316cc#diff-a75e9754db9269401928d444c9f036bc068af538ee5ca551e761767a5394e23eR339).